extensions/rich_text_components/components_test.py

```diff
@@ -120,14 +120,30 @@ def test_link_validation(self):
     def test_math_validation(self):
         """Tests collapsible component validation."""
         valid_items = [{
-            'raw_latex-with-value': '123456789'
+            'math_content-with-value': {
+                u'raw_latex': u'123456',
+                u'svg_filename': u''
+            }
         }, {
-            'raw_latex-with-value': '\\frac{x}{y}'
+            'math_content-with-value': {
+                u'raw_latex': u'\\frac{x}{y}',
+                u'svg_filename': u''
```

Comment:
Create an issue to have a validation for svg_filename in future and add a TODO here!

Reply:
done